dcc-portal-ui/app/scripts/ui/js/table.js

```diff
@@ -238,6 +244,24 @@ angular.module('icgc.ui.table.pagination', [])
         };
       }
     };
+  })
+  .directive('paginationClientSide', function(){
```

Comment:
Would prefer `pagedTable/List` or `paginatedTable/List` or  as the directive name, thoughts, @btiernay ?
IIRC whether the pagination is done client side or server side can be determined by options passed into the library
Speaking of which, did we do a scan first for existing libraries? 
http://ng-table.com/
^ This one has 12,603 downloads from npm last month (that doesn't include bower downloads), and 2.5k stars on Github
Might have more features and end up requiring us to add less code